config/install/migrate_plus.migration.node.yml

```diff
@@ -40,29 +39,78 @@ process:
   # in.
   field_alternative_title:
     plugin: skip_on_empty
-    source: subtitle 
+    source: subtitle
     method: process
 
   field_description: description
 
   # Dates are EDTF strings
   field_edtf_date: issued
-    
-  # Make the object an 'Image'
+
+  _mimetype:
+    -
+      # Get a mimetype (eg image/jpeg) from mimetype process plugin.
+      source: file
+      plugin: file_mimetype
+
+  _model:
+    -
+      # Get a mimetype (eg image/jpeg) from mimetype process plugin.
+      source: file
+      plugin: file_mimetype
+    -
+      # Get the type only (strip slash and subtype).
+      plugin: str_replace
+      source: '@_mimetype'
+      search: "#/.*#"
+      replace: ""
+      regex: true
+    -
+      # Our models are: Audio, Binary, Image, Video.
+      # Convert anything not in that list to "binary". (Even "text"?)
+      plugin: static_map
+      map:
+        audio: Audio
+        image: Image
+        video: Video
+      default_value: Binary
+
+  # Extract model from the supplied filename.
   field_model:
-    plugin: entity_lookup
-    source: constants/model
-    entity_type: taxonomy_term
-    value_key: name 
-    bundle_key: vid
-    bundle: islandora_models 
+    -
+      # Get a mimetype (eg image/jpeg) from mimetype process plugin.
+      source: file
+      plugin: file_mimetype
```

Comment:
Same question, I'm guessing you added `_mimetype` and made the change and just forgot to remove these actions.